src/bilder/images/concourse/deploy.py

```diff
@@ -84,24 +103,24 @@
         partial(
             VaultTemplate,
             contents=(
-                "{{ with secret 'secret-concourse/tsa_key' }}"
-                "{{ .Data.data.private_key }}{{ end }}"
+                '{{ with secret "secret-concourse/web" }}'
+                "{{ printf .Data.data.tsa_private_key }}{{ end }}"
```

Comment:
Why `printf`?

Reply:
Because otherwise the newlines are left as literal `\n` instead of being rendered